xbmc/Application.cpp

```diff
@@ -4611,7 +4611,7 @@ void CApplication::ProcessSlow()
 
   // Temporarely pause pausable jobs when viewing video/picture
   int currentWindow = g_windowManager.GetActiveWindow();
-  if (CurrentFileItem().IsVideo() || CurrentFileItem().IsPicture() || currentWindow == WINDOW_FULLSCREEN_VIDEO || currentWindow == WINDOW_SLIDESHOW)
+  if (currentWindow == WINDOW_FULLSCREEN_VIDEO || currentWindow == WINDOW_SLIDESHOW)
```

Comment:
this would have other negative effects scenarios like TV in minimized window. If video plays in the background even more resources are required and the need to pause those kind of jobs is even higher.